src/openAi.ts

```diff
@@ -5,7 +5,7 @@ const configuration = new Configuration({
 });
```

Comment:
GPT summary of [5705bd](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/c8c47408c482f791430fa4be8b1253005844ef70/src/openAi.ts#5705bd8316a205044296b8f3a09751754be6998e) - [b817c5](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/14bc7d58520d55fecaa099a6e45930d84b7be731/src/openAi.ts#b817c578834ff03d5d8c6b9610ef7f67d4f95e39):
* Changed the model name from "text-davinci-003" to "text-chat-davinci-002-20230126"
* Updated the maximum open AI query length to 20000
* Updated the temperature to 0.5
* Updated the maximum tokens to 512